### PR TITLE
Form register for next edition

### PR DESCRIPTION
**Pull request by carlotacb**

### CODE REVIEW

**Review by mauriciabad — COMMENTED** [review]
https://github.com/hackupc/hackupc-landing/pull/623

**Review by mauriciabad — APPROVED** [review]

**Review by EncryptEx — APPROVED** [review]
Perfect, looks utterly awesome.

**Comment by mauriciabad** [comment]
I just remembered that would be nice to send a Google analytics event when the form is submitted

**Comment by carlotacb** [comment]
> I just remembered that would be nice to send a Google analytics event when the form is submitted

Can you added please? @mauriciabad I agree it's a good idea!

File: src/styles/sections/hero/hero.scss [hunk 1]
```diff
@@ -85,3 +85,76 @@ $fade-height: 200px;
...
+      padding-left: 8px;
+    }
+    &-input {
+      display: block;
+      width: 100%;
+      box-sizing: border-box;
+      padding: 10px 20px;
+      border: 3px solid #3f5081;
+      backdrop-filter: blur(2px);
+      background-color: rgb(255 255 255 / 67%);
```
Comment (EncryptEx) [inline src/styles/sections/hero/hero.scss]: Email Input in solid white would be a great improvement. (As we accorded)
```suggestion
      background-color: rgb(255 255 255);
```

**Review by EncryptEx — CHANGES_REQUESTED** [review]

File: src/styles/sections/hero/hero.scss (hunk 1 above)
Comment (mauriciabad) [reply src/styles/sections/hero/hero.scss]: Done

**Review by EncryptEx — COMMENTED** [review]
All perfect

**Pull request merged**